debugging.rst

```diff
@@ -0,0 +1,84 @@
+.. _debugging:
+
+==============
+Debugging Jobs 
+==============
+
+Flux supports parallel debuggers such as Rogue Wave Software (RWS)'s
+TotalView parallel debugger (https://www.roguewave.com).
+
+----------------------------------
+Parallel Debugging using TotalView
+----------------------------------
+
+Debugging your MPI job from the beginning of
+parallel program execution:
+
+.. code-block:: sh
+
+  totalview --args flux mini run -N 2 -n 2 ./mpi-program
+
+Attaching to an already running job:
+
+.. code-block:: sh
+
+  totalview --args flux mini attach <JOBID>
+
+Note that you should use ``totalview --args flux mini attach`` whether your
```

Comment:
ReSt has a note directive, if you want to use that:

```suggestion
.. note::
You should use ``totalview --args flux mini attach`` whether your
```

Reply:
Cool!